lms/static/sass/features/_course-experience.scss

```diff
@@ -171,3 +171,21 @@
     }
   }
 }
+
+// Course Updates Page
+div.course-updates {
```

Comment:
FYI we don't include element names in our Sass, so this would be just ``.course-updates {``